app/controllers/sessions_controller.rb

```diff
@@ -1,2 +1,30 @@
 class SessionsController < ApplicationController
+
+  def new
+  end
+
+  def create
+    @user = User.find_by(email: params[:session][:email])
+
+    unless @user.nil?
```

Comment:
I almost did it this way, but if I wanted to keep the different flash messages I had to do the check separately. If we just want one generic flash message, I can change it.